active/package-maintainers.md

```diff
@@ -0,0 +1,126 @@
+- Collaborative documents set up in the Django Software Foundation’s Google Workspace.
+- A new private #package-maintainers-wg` channel on the Django Discord server for internal discussions.
+
+The group will also offer office hours for package maintainers every month via a video call.
+
+## Reporting
+
+The group will share highlights from current activities on a monthly basis via a forum post.
+
+## Appendix
+
+### Group activities
+
+As an illustration of the group’s remit, here are possible activities members could take part in.
+
+#### Support Django version compatibility efforts
+
+As Django regularly releases new versions, it’s an ongoing effort to ensure packages are compatible with new releases.
+
+Group members can support maintainers in this effort by:
+
+- Creating a shared calendar with Django release dates and deadlines.
+- Creating communication channels for package maintainers to coordinate compatibility changes for a specific release.
+- Recommending appropriate automation, such as [django-upgrade](https://github.com/adamchainz/django-upgrade) or how to set up Django pre-releases in Continuous Integration.
+- Proposing additions to the Django release notes with recommendations on how to handle deprecations.
+
+#### Share packaging best practices
+
+The group can work on create, maintain, or curate best practices for Django package maintainers. For example, the official [Advanced tutorial: How to write reusable apps](https://docs.djangoproject.com/en/5.0/intro/reusable-apps/), or packaging-focused sections on third-party resources like [awesome-django](https://github.com/wsvincent/awesome-django).
+
+Those best practices can cover aspects like:
+
+- Documentation approaches and tools
+- Use of the `django_` prefix for app names (see [Change reusable apps naming recommendation](https://forum.djangoproject.com/t/change-reusable-apps-naming-recommendation/25233)).
+- Recommended Python, Django, browser, operating system support policies to align with those of Django itself.
+- Tooling, for example how to set up Continuous Integration for Django packages.
+
+#### Facilitate community-driven maintenance
+
+Group members can liaise with organizations dedicated to sharing maintenance efforts, such as [Jazzband](https://jazzband.co/), [Wagtail Nest](https://github.com/wagtail-nest), or [Django Commons](https://github.com/django-commons). Or work directly with packages in need of new maintainers to find new candidates. This can involve:
+
+- Sharing calls for new package maintainers
+- Coordinating trusted volunteer open source “roadies” familiar with package ownership transfers.
+- Facilitating participation in mentoring programs such as [Google Summer of Code](https://summerofcode.withgoogle.com/) or [Djangonaut Space](https://djangonaut.space/).
+- Providing support for tasks requiring very specific expertise, like funding avenues or vulnerability reports handling.
```

Comment:
Some further ideas for what the "Facilitate community-driven maintenance" activity could include:

* **A shared blocklist, or at least a warning flag.** Every maintainer seems to have to learn for themselves about particular users who consistently file annoying, useless bugs and do not otherwise productively contribute. Could there be a shared blocklist to help with this?
* **Boilerplate policies.** Beyond licenses and codes of conduct, maintainers could use a shared set of customizable policies on deprecation, security reporting, release announcement, vendoring, user support, contributing guidelines, testing, architectural change, and data privacy. The WG could also share new issue templates, and replies to reuse for common misunderstandings (like "you'll need to rebase" or "we're not able to prioritize this right now but a patch or a donation could change that").
* **Shared user experience research efforts.**  Pool funds and invest in UX research for a suite of tools that people often use together, and learn surprising and helpful things that help all package maintainers improve their projects' UX. ([Example: what pip did.](https://harihareswara.net/posts/2024/model-ux-research-design-docs-for-command-line-open-source/))
 
(I further discuss these ideas in [this blog post](https://www.harihareswara.net/posts/2023/user-support-equanimity-potential-cross-project-tools-practices-open-source/).)

Reply:
I'm not a big fan of blocklists or warning flags for contributors. Maintaining and growing community is one of the biggest challenges for open source communities in my experience. New participants might be learning or may genuinely be a troll, but even in those cases they are engaging. Rather than block them I'd lean toward a policy of educating and including them.  I'd rather get a bug report than not, even if it's not a real bug it may be a UX design issue. Even if the reporter is upset and it's coming through in their tone, I'd rather get the bug report than not. I can choose to ignore people's tone and frustration. Although I do agree there need to be rules in regard to abusive language and personal attacks... but those should be listed up front and moderation steps clearly outlined and applied consistently.